securesystemslib/gpg/common.py

```diff
@@ -647,7 +650,7 @@ def parse_signature_packet(data, supported_signature_types=None,
   if signature_algorithm not in SUPPORTED_SIGNATURE_ALGORITHMS:
     raise ValueError("Signature algorithm '{}' not "
         "supported, please verify that your gpg configuration is creating "
-        "either DSA or RSA signatures (see RFC4880 9.1. Public-Key "
+        "either DSA, RSA, or EdDSA signatures (see RFC4880 9.1. Public-Key "
```

Comment:
Thanks for spotting this.  There's a similar error message that needs updating in `parse_pubkey_payload`.